components/inputs/test/input-time.test.js

```diff
@@ -0,0 +1,130 @@
+import '../input-time.js';
+import { aTimeout, expect, fixture, oneEvent } from '@open-wc/testing';
+
+const basicFixture = '<d2l-input-time label="label text"></d2l-input-time>';
+
+describe('d2l-input-time', () => {
+
+	let elem;
+
+	beforeEach(async() => {
+		elem = await fixture(basicFixture);
+	});
+
+	function getInput() {
+		return elem.shadowRoot.querySelector('.d2l-input');
+	}
+
+	describe('accessibility', () => {
+
+		it('passes all axe tests', () => {
+			expect(elem).to.be.accessible();
```

Comment:
You actually need an `await` before the expect for all these -- otherwise the test returns and passes.

Reply:
Still missing `await`s before each of the `expect`s.

Reply:
Recap of elsewhere conversation: For some reason VScode warns that adding `await` will have no effect, but actual testing and the [Chai A11y test docs for open-wc](https://open-wc.org/testing/testing-chai-a11y-axe.html) prove it does. Making this change in next commit 👍